CHANGELOG.md

```diff
@@ -1,23 +1,25 @@
 # New Relic Ruby Agent Release Notes
 
 
-## dev 
+## v9.7.1
```

Comment:
jsyk, you don't actually need to update the version number, the prerelease automation will do it automatically. No need to change it back though! the automation won't break if its already changed.

Reply:
I didn't know this! Very good to know ◡̈